elements/pl-fingerprint/info.json

```diff
@@ -0,0 +1,11 @@
+{
+    "controller": "pl-fingerprint.py",
+    "dependencies": {
+        "elementStyles": [
+            "pl-fingerprint.css"
+        ],
+        "elementScripts": [
+            "pl-fingerprint.js"
+        ]
+    }
+}
```

Comment:
Is all this `pl-fingerprint` stuff supposed to be in this PR?

Reply:
oops, no... `pl-fingerprint` is something different that I worked on shortly for one of my profs... since it was quite urgent I didn't have time to branch properly and left the code in here :sweat_smile:, I'll be cleaning all the irrelevant code out shortly